builtin/client/chatcommands.lua

```diff
@@ -23,6 +23,14 @@ core.register_on_sending_chat_message(function(message)
 		return true
 	end
 
+	-- Check registered event callbacks
+	for _, func in pairs(core.registered_on_chatcommand) do
+		-- if callback returns true, default handlers should not be used
+		if func(cmd, param) == true then
+			return true
+		end
+	end
```

Comment:
Use `core.run_callbacks(core.registered_on_chatcommand, 5, cmd, param)`.
This is mode 5:
https://github.com/minetest/minetest/blob/eef62c82a2e58700fc1216b0b8c03e421bc77995/src/script/common/c_internal.h#L94-L96